internal/cloud/cloud_username_manager.go

```diff
@@ -91,7 +91,7 @@ func (c *CloudUsernameManager) CheckUsername(ctx context.Context, st store.RStor
 		return
 	}
 
-	req, err := http.NewRequest("GET", u.String(), body)
```

Comment:
do we also need to update TFT to handle POSTs? Currently it's only setup to handle GET
https://github.com/windmilleng/tft/blob/master/snapshot_frontend/server/server.go#L106

Reply:
Yes, that's in a forthcoming PR. Sorry, I should have made that explicit in the description.